applications/plugins/SofaCarving/CarvingManager.cpp

```diff
@@ -76,11 +76,7 @@ void CarvingManager::init()
 {
     // Search for collision model corresponding to the tool.
     if (d_toolModelPath.getValue().empty())
-    {
         m_toolCollisionModel = getContext()->get<core::CollisionModel>(core::objectmodel::Tag("CarvingTool"), core::objectmodel::BaseContext::SearchDown);
```

Comment:
I didn't know that it was possible to retrieve objects in ctx with tags. is it possible to get rid of the getContexts here?

Reply:
no idea.